cl/search/documents.py

```diff
@@ -539,3 +547,276 @@ def prepare_person_child(self, instance):
 
     def prepare_school(self, instance):
         return [e.school.name for e in instance.educations.all()] or None
+
+
+# RECAP
+class DocketBaseDocument(Document):
+    docket_child = JoinField(relations={"docket": ["recap_document"]})
+    timestamp = fields.DateField()
+
+    class Django:
+        model = Docket
+        ignore_signals = True
+
+    def prepare_timestamp(self, instance):
+        return datetime.utcnow()
+
+
+@recap_index.document
+class RECAPDocument(DocketBaseDocument):
+    id = fields.IntegerField(attr="pk")
+    docket_entry_id = fields.IntegerField(attr="docket_entry.pk")
+    description = fields.TextField(
+        attr="docket_entry.description",
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(
+                attr="docket_entry.description", analyzer="english_exact"
+            ),
+        },
+        search_analyzer="search_analyzer",
+    )
+    entry_number = fields.IntegerField(attr="docket_entry.entry_number")
```

Comment:
I'm not sure we need this if we also have document_number, right? Aren't they always the same thing, and didn't we omit this field in solr?

Reply:
Yeah, the `document_number` is the same that the `entry_number`. Both of them are currently present in the Solr version.

An both are displayed in the API.
![Screenshot 2023-09-13 at 15 00 29](https://github.com/freelawproject/courtlistener/assets/486004/c5c5c522-081a-4f5a-8e5e-9cffc56d49fb)

So that is it ok to remove it? Some users can be using `entry_number`.

Reply:
OK, I suppose we better keep it. :)